src/components/controls/ButtonCopy.vue

```diff
@@ -0,0 +1,64 @@
+<template>
```

Comment:
unsure of naming conventions, but seems like CopyButton would be a more natural name